### ISSUE TITLE
Clarification on Retrieval Metrics

### ISSUE THREAD

**Issue by varadgunjal** [ABaldrati/CLIP4Cir#8]
@ABaldrati 

Thank you for the great work!

I'm just starting out surveying image retrieval literature for usage with pretrained vision-language models and had a clarification question regarding the evaluation metrics.

Recall_subset@K from CIRR makes sense to me because the total number of relevant items is capped at the number of elements in the query image's subset. My question relates to the Recall@1, 10, 50 numbers reported here (and in other literature). How were these numbers calculated? What is the value of the denominator when calculating Recall since we don't know the true number of False negatives?

If I understand the code correctly here -  https://github.com/ABaldrati/CLIP4Cir/blob/2b9fc39a0269fff1ecffab4289b4ffaf72aa07eb/src/validate.py#L54

where `len(labels_ == len(target_names)` and `target_names` is just the list of labels of the validation set, I gather that the denominator is simply the count of all images in the validation set. Am I correct in saying that?
